gen-apidocs/config/v1_30/config.yaml

```diff
@@ -330,21 +334,18 @@ operation_categories:
       match: connect${group}${version}Get(Namespaced)?${resource}Attach
     - name: Create Connect Attach
       match: connect${group}${version}Post(Namespaced)?${resource}Attach
+    - name: Replace Namespace Finalize
+      match: replace${group}${version}(Namespaced)?${resource}Finalize
+    - name: Read Approval
+      match: read${group}${version}(Namespaced)?${resource}Approval
+    - name: Patch Approval
+      match: patch${group}${version}(Namespaced)?${resource}Approval
+    - name: Replace Approval
+      match: replace${group}${version}(Namespaced)?${resource}Approval
 
 # List of *partial* operation IDs for matching. All matched operations are
 # excluded from the reference doc.
 excluded_operations:
-  - connectCoreV1Patch
-  - createCoreV1NamespacedPodBinding
-  - getCodeVersion
-  - logFileHandler
-  - logFileListHandler
-  - replaceCoreV1NamespaceFinalize
-  - V1beta1CertificateSigningRequestApproval
-  - V1CertificateSigningRequestApproval
-  - V1beta1NamespacedReplicationControllerDummyScale
-  - getServiceAccountIssuerOpenIDConfiguration
-  - getServiceAccountIssuerOpenIDKeyset
 
```

Comment:
Are these removals safe?

Reply:
The tool will emit log lines but those operations do not have an associated definition, so no documentation is generated for them and the documentation doesn't change. Feel free to verify.

Reply:
So ... this should be based on #363, right?

Reply:
I tested it both with and without #363. Without it, the endpoints that have new regex expressions are added and those that do not have a definition emit a message and do not get generated. With it, all previously excluded endpoints have documentation. It makes more sense after the previous commit but it wouldn't cause errors without it also be fine without it. I could merge them into a single PR if that makes more sense